base/deprecated.jl

```diff
@@ -1076,4 +1076,9 @@ eval(Base.LinAlg, quote
     end
 end)
 
+
+@deprecate stop_reading(stream::LibuvStream) pause_reading(stream)
```

Comment:
Upon reflection, renaming this is unnecessary. I can just slightly change how it is implemented to improve performance on Unix.